ui/src/pages/ShoeMap.tsx

```diff
@@ -80,7 +80,7 @@ export const ShoeMap: React.FC = () => {
   const currentLocation = { lat: 43.4723, lng: -80.5449 };
 
   const { data: stories, error } = useSWR<Story[]>("/api/stories");
-
+  const [filteredStories, setFilteredStories] = useState([]);
```

Comment:
This is not an array of stories, it's an array of countries, so we should probably rename accordingly.